Engine/Client/Src/Network/NetworkClient.hpp

```diff
@@ -16,6 +16,8 @@
 #include "PlayerComponent.hpp"
 #include "ClientSceneManager.hpp"
 
+# define POSITION_PLAYER_ENTITY_SERVER 8
```

Comment:
Where does it is used ? (I didn't see it) And I don't think this is a great idea since it'll change as soon as we add another systems before the player systems in the list